app/controllers/orders_controller.rb

```diff
@@ -29,8 +29,12 @@ def destroy
 
   def update
     @order = get_order
-    @order.update(order_params)
-    redirect_to confirm_order_path
+    if @order.update(order_params)
+      redirect_to confirm_order_path
+    else
+      flash[:alert] = 'Error then saving order!'
```

Comment:
Error WHEN saving order. Do we have a test for this `else`block?  Is there a use case`